src/Illuminate/Encryption/Encrypter.php

```diff
@@ -112,10 +112,10 @@ public function decrypt($payload)
      *
      * @throws \Exception
      */
-    protected function mcryptDecrypt($value, $iv)
+    protected function opensslDecrypt($value, $iv)
```

Comment:
We already have a public decrypt method.

I vouch for just merging the two methods for encryption and decryption.